src/content/en/updates/2020/06/devtools.md

```diff
@@ -13,6 +13,12 @@ description: Style editing for CSS-in-JS frameworks, Lighthouse 6.0, new JavaScr
 
 {% include "web/_shared/contributors/jecelynyeen.html" %}
 
+<div class="video-wrapper-full-width">
+  <iframe class="devsite-embedded-youtube-video" data-video-id="2EiPb1opH3g"
```

Comment:
This is the Chrome 85 blog post, but the ID refers to the video for Chrome 80: https://www.youtube.com/watch?v=2EiPb1opH3g  Is this correct?

Reply:
Ah, I forget to leave a comment. No, the video is almost ready, and will be ready by today. This is a placeholder to get approval (I will update the id when it's available) beforehand.

Reply:
I see, thanks for clarifying. LGTM then :)